docs/gathered-data.md

```diff
@@ -747,29 +748,8 @@ None
 
 ## ContainersLogs
 
```

Comment:
I think this change was 'inherited' by another change where it didn't happen the docs update. This file was modified using the project's make target automatically.

Reply:
yes the change below is my incomplete work on documentation :-/ so it should be improved, but we can do it in different PR